### ISSUE TITLE
sshd_config_maxstartups - how to define

### ISSUE THREAD

**Issue by kkruzich** [ghoneycutt/puppet-module-ssh#251]
None of these work:

```
  # sshd_config_maxstartups => 3:50:10,
  # sshd_config_maxstartups => '3:50:10',
  # sshd_config_maxstartups => "3:50:10",
```
error is: 

`ssh::sshd_config_maxstartups may be either an integer or three integers separated with colons, such as 10:30:100. Detected value is <3:50:10>`

**Comment by ghoneycutt** [comment]
Thanks @kkruzich 

Seems there was a problem in the regex for an integer with only one digit. I've created PR #252 to address this.

**Comment by ghoneycutt** [comment]
Released in v3.56.1

**Comment by kkruzich** [comment]
Fantastic! Thank you!
